modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -117,6 +117,8 @@
 public class ItNodeTest {
     private static final IgniteLogger LOG = IgniteLogger.forClass(ItNodeTest.class);
 
+    private static final int ELECTION_TIMEOUT_MILLIS = 1000;
```

Comment:
We already have org.apache.ignite.raft.jraft.core.TestCluster#ELECTION_TIMEOUT_MILLIS for a default value.
No need to create another.

Reply:
OK. Made org.apache.ignite.raft.jraft.core.TestCluster#ELECTION_TIMEOUT_MILLIS public.